VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppRecruitmentScreen/InAppRecruitmentScreen.tsx

```diff
@@ -53,6 +77,7 @@ const InAppRecruitmentScreen: FC<InAppRecruitmentScreenProps> = ({ navigation })
             numberOrUrlLink={LINK_URL_VETERAN_USABILITY_PROJECT}
             linkType={LinkTypeOptionsConstants.url}
             a11yLabel={t('inAppRecruitment.learnMore')}
+            fireAnalytic={() => logAnalyticsEvent(Events.vama_givefb_info())}
```

Comment:
I wonder if we should change this analytic to a generic and apply it to more sections than just this with some parameters to differentiate the two